lib/charms/finos_legend_libs/v0/legend_operator_base.py

```diff
@@ -25,7 +25,7 @@
 
 # Increment this PATCH version before using `charmcraft publish-lib` or reset
 # to 0 if you are raising the major API version
-LIBPATCH = 6
+LIBPATCH = 5
```

Comment:
You may need to bump this - I'm not sure we can "unpublish" version 6 from the store?

Reply:
We didn't publish version 6, so this is safe to revert:

```
ubuntu@ubuntu:~/workdir/legend-juju-engine-server-operator$ charmcraft fetch-lib charms.finos_legend_libs.v0.legend_operator_base
Library charms.finos_legend_libs.v0.legend_operator_base updated to version 0.5.
ubuntu@ubuntu:~/workdir/legend-juju-engine-server-operator$ charmcraft fetch-lib charms.finos_legend_libs.v0.legend_operator_testing
Library charms.finos_legend_libs.v0.legend_operator_testing updated to version 0.6.
```